Jenkinsfile

```diff
@@ -81,7 +77,8 @@ pipeline {
                   script {
                     def dockerRepoName = 'zooniverse/fe-project'
                     def dockerImageName = "${dockerRepoName}:${GIT_COMMIT}"
-                    def newImage = docker.build(dockerImageName)
+                    def buildArgs = "--build-arg ASSET_PREFIX SENTRY_DSN ."
```

Comment:
Same question here about build args syntax.

Reply:
yep - correct. they have to be specified each time.